specification/cognitiveservices/OpenAI.Inference/models/completions/chat_completions.tsp

```diff
@@ -77,20 +77,17 @@ model ChatCompletionsJsonSchemaResponseFormat
   };
 }
 
-// Deferred to a near-term future release after publish; see:
-// https://github.com/Azure/azure-rest-api-specs/pull/30087
-//
-// /**
-//  * Options for streaming response. Only set this when you set `stream: true`.
-//  */
-// @added(ServiceApiVersions.v2024_07_01_Preview)
-// model ChatCompletionStreamOptions {
-//   /**
-//    * If set, an additional chunk will be streamed before the `data: [DONE]` message. The `usage` field on this chunk shows the token usage statistics for the entire request, and the `choices` field will always be an empty array. All other chunks will also include a `usage` field, but with a null value.
-//    */
-//   #suppress "@azure-tools/typespec-azure-core/casing-style" "OpenAI uses wire-format snake_casing"
```

Comment:
Was there a reason for using `#suppress` versus the update to `@encodedName` ?

Reply:
Either works; I've migrated towards using `suppress` for this in the layered spec given we should probably just suppress it across the board, but there's a lot to be said for internal consistency.